.github/workflows/ci.yml

```diff
@@ -16,12 +16,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Setup JDK
-        uses: rtyley/setup-java@patch-1
-        with:
-          distribution: corretto
-          java-version: 11
-          cache: sbt
+      - name: Setup Java and sbt
+        uses: guardian/setup-scala@v1
```

Comment:
looks like the existing .tools-versions is set to java 21  instead of 11

Reply:
https://github.com/guardian/guardian-configuration/blob/cc7c03eb858ba630d62e9e80789ef0b7f890815f/.tool-versions#L1

...ah yeah - so this project will now _build_ with Java 21, rather than 11. However, it is also set to remain _compatible_ with Java 8: 

https://github.com/guardian/guardian-configuration/blob/cc7c03eb858ba630d62e9e80789ef0b7f890815f/build.sbt#L43

...so it can definitely be consumed by all current projects at the Guardian (which should be [aiming to run on Java 21](https://github.com/guardian/maintaining-scala-projects/issues/8) in any case!).